### DISCUSSION TITLE
银行卡号识别，配置文件的版本越高，准确率越低？

### DISCUSSION THREAD

**Discussion by minushuang** [PaddlePaddle/PaddleOCR#12362] [Q&A]
做银行卡识别
配置文件PP-OCRv4/ch_PP-OCRv4_rec.yml，acc 50%；
配置文件PP-OCRv3/en_PP-OCRv3_rec.yml, acc 80%;
配置文件ch_ppocr_v2.0/rec_chinese_lite_train_v2.0.yml, acc 93%(中间结果，还在训练中)
这几个配置文件，算法不同，backbone不同，loss不同。

是不是表示卡号识别场景rcnn+ctc更加适配一些？

**Comment by tink2123** [comment]
如果训练数据较少的话，配置越高越容易过拟合。
